src/pages/faq.jsx

```diff
@@ -0,0 +1,104 @@
+          either use our hosted Zeno Hub or host it yourself. We welcome contributions from the
+          community to various aspects of Zeno, including:
+        </p>
+        <ul>
+          <li>
+            <a href="https://github.com/zeno-ml/zeno-hub">Zeno Hub</a> - Core Zeno platform code
+          </li>
+          <li>
+            <a href="https://github.com/zeno-ml/zeno-instance-views">Zeno Instance Views</a> -
+            Specification and rendering for custom data views.
+          </li>
+          <li>
+            <a href="https://github.com/zeno-ml/zeno-build">Zeno Build</a> - Repository of example
+            evaluations
+          </li>
+        </ul>
+        <h3>How do I get started?</h3>
+        <p>
+          You can start by{" "}
+          <a href="https://hub.zenoml.com/home/cabreraalex">
+            exploring public projects and reports
+          </a>{" "}
+          to get an idea of how Zeno works. To create your own projects please follow the{" "}
+          <a href="/docs/intro#creating-a-project">Getting Started Guide</a>{" "}
+        </p>
+        <h3>How do I create a new project?</h3>
+        <p>
+          Check out the <a href="/docs/intro#creating-a-project">Getting Started Guide</a> to learn
+          how to use the <code>zeno-client</code> Python library to create projects, upload data,
+          and upload AI system outputs.
+        </p>
+        <h3>How do I get data for evaluation?</h3>
+        <p>
+          Zeno does not provide evaluation data. We recommend you start with a dataset similar to
+          your task from public repositories such as{" "}
+          <a href="https://github.com/huggingface/datasets">🤗 datasets</a>, gather data from users
+          or customers, or bootstrap an evaluation set using a model or system such as ChatGPT.
+        </p>
+        <h3>How do I get model outputs?</h3>
+        <p>
+          Zeno also does not provide or run AI systems. Run models locally using services such as{" "}
+          <a href="https://huggingface.co/docs/transformers/index">🤗 transformers</a> and upload
+          the results to Zeno.
+        </p>
+        <h3>Can I run Zeno locally?</h3>
+        <p>
+          Check out the{" "}
+          <a href="https://github.com/zeno-ml/zeno-hub/blob/main/DEVELOPMENT.md">
+            Zeno Hub Developer documentation
+          </a>{" "}
+          for instructions on how to run the Docker containers and host Zeno yourself.
+        </p>
+        <h3>How can I report an issue with Zeno?</h3>
+        <p>
+          Please send us a message on our{" "}
+          <a href="https://discord.gg/km62pDKAkE">Discord channel</a>, file a bug on{" "}
+          <a href="https://github.com/zeno-ml/zeno-hub/issues">GitHub</a>, or email{" "}
+          <a href="mailto:hello@zenoml.com">
+            <code>hello@zenoml.com</code>
+          </a>{" "}
+          if you run into any issues!
+        </p>
```

Comment:
Maybe move this up? Don't know though fine as is as well.

Reply:
done!